package.json

```diff
@@ -43,5 +43,17 @@
   },
   "engines": {
     "node": ">= 0.10"
+  },
+  "keywords": [
+    "hypernova",
+    "client"
+  ],
+  "directories": {
```

Comment:
Well, some tools reference it … but yeah, it's not really widespread yet. Kind of a catch-22 …